.rubocop_todo.yml

```diff
@@ -297,6 +297,7 @@ Metrics/AbcSize:
     - 'app/controllers/api/v2/transfers_controller.rb'
     - 'app/jobs/export_pool_xp_to_traction_job.rb'
     - 'app/sample_manifest_excel/sample_manifest_excel/manifest_type_list.rb'
+    - 'spec/models/bulk_submission_scrna_spec.rb'
```

Comment:
try not to add more rubocop todos?

Reply:
Yes, although do we care if it's in a test? I'm not going to attempt to change it now as it's probably not trivial to fix (AbcSize) and I want to get these out.